extensions/2.0/Vendor/EXT_instance_features/README.md

```diff
@@ -61,15 +65,57 @@ When the feature ID definition does not refer to an instance attribute, then the
 > }
 > ```
 
-## Optional vs. Required
+A feature ID set may also include the following properties:
 
-This extension is optional, meaning it should be placed in the `extensionsUsed` list, but not in the `extensionsRequired` list.
+* `nullFeatureId`: a value that indicates that a certain instance is not considered to be an identifiable object
+* `label`: an alphanumeric string used to identify feature ID sets across different glTF nodes
+* `propertyTable`: the index of a property table in the [`EXT_structural_metadata`](../EXT_structural_metadata/) extension
+
+> **Example:** A more complex example with two feature ID sets. The first feature ID set groups instanced trees into forests. The tree with feature ID `2` matches the `nullFeatureId` and therefore does not belong to a forest feature. The second feature ID set (feature ID values not shown) is defined implicitly based on instance index, and identifies individual trees. Each feature ID set refers to a property table that contains metadata about the features.
+>
+> ![Feature ID by instance](figures/trees.png)
+>
+> ```jsonc
+> {
+>   "nodes": [
+>     {
+>       "mesh": 0,
+>       "extensions": {
+>         "EXT_mesh_gpu_instancing": {
+>           "attributes": {
+>             "TRANSLATION": 0,
+>             "ROTATION": 1,
+>             "SCALE": 2,
+>             "_FEATURE_ID_0": 3
+>           },
+>         },
+>         "EXT_instance_features": {
+>           "featureIds": [
+>             {
+>               "nullFeatureId": 2,
+>               "featureCount": 2,
+>               "attribute": 0,
+>               "propertyTable": 0,
+>               "label": "Forests"
+>             },
+>             {
+>               "featureCount": 9,
+>               "propertyTable": 1,
+>               "label": "Trees"
+>             }
+>           ]
+>         }
+>       }
+>     }
+>   ]
+> }
+> ```
```

Comment:
Added a second example to show other concepts like grouping instances into features and defining implicit feature IDs